packages/sommelier.rb

```diff
@@ -111,21 +103,30 @@ def self.install
   def self.postinstall
     puts '', <<~RESTARTSOMMELIER_EOT.orange
       To complete the installation, execute the following:
-      source #{CREW_PREFIX}/etc/profile
-      restartsommelier
+
+        source #{CREW_PREFIX}/etc/profile
+        restartsommelier
     RESTARTSOMMELIER_EOT
 
     puts <<~ENV_ADJUSTMENT_EOT.lightblue
       To adjust sommelier environment variables, edit #{CREW_PREFIX}/etc/env.d/sommelier
       Default values are in #{CREW_PREFIX}/etc/env.d/sommelier
 
-      Run `startsommelier` to start sommelier daemon.
-      Run `stopsommelier` to stop all sommelier daemon.
-      Run `restartsommelier` to restart sommelier daemon
+      Run 'startsommelier' to start sommelier daemon.
+      Run 'stopsommelier' to stop all sommelier daemon.
+      Run 'restartsommelier' to restart sommelier daemon
     ENV_ADJUSTMENT_EOT
 
     puts <<~GUI_WARNING_EOT.orange
       Please be aware that GUI applications may not work without the sommelier daemon running.
     GUI_WARNING_EOT
+
+    puts <<~DPI_EOT.lightcyan
+      If the UI scale does not match the Chrome OS browser settings, execute the following:
+
+        echo 'SOMMELIER_APPLY_DPI_FIX=1' >> #{CREW_PREFIX}/etc/env.d/sommelier
+        stopsommelier
+        source #{CREW_PREFIX}/etc/profile
```

Comment:
Why not just do this anyway without a message?

Reply:
As this bug seems only appears on my system...